opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/BaseInteractor.java

```diff
@@ -150,6 +151,7 @@ public BaseInteractor(BasePresenter presenterCallBack) {
         taskUtils = TaskUtils.getInstance();
         database = getInstance().getRepository().getReadableDatabase();
         prefsUtil = PreferencesUtil.getInstance();
+        revealApplication = RevealApplication.getInstance();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'RevealApplication.getInstance' due to existing static import 'org.smartregister.reveal.application.RevealApplication.getInstance'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5564566)